### ISSUE TITLE
How handle desktop windows using qaf.

### ISSUE THREAD

**Issue by kurawadiprasanna** [qmetry/qaf#392]
### QAF Version
> Note: only the latest version is supported

### Steps To Reproduce
    1.
    2.
    3.

### Is the issue reproducible on runner?

- [x] QAS
- [ ] Maven
- [ ] Gradle
- [x] Ant
- [ ] Eclipse

### Test case sample
Not sure, how to integrate Auto it or robot framework to QAF. Please advise

**Comment by cjayswal** [comment]
This is not right place for Q & A. It is recommended to post questions with details in [users group](https://groups.google.com/g/qaf-users) or in [stack-overflow](https://stackoverflow.com/questions/tagged/qaf) with qaf tag.
